documentation/api/v1/session_event_api_doc.rb

```diff
@@ -32,6 +36,35 @@ module SessionEventApiDoc
       key :description, 'Return session events that are stored in the database.'
       key :tags, [ 'session_event' ]
 
+      parameter do
+        key :name, :limit
+        key :in, :query
+        key :description, 'The maximum number of session events that will be retrieved from the query. (Default: 100)'
+        key :example, 100
+        key :type, :integer
+        key :format, :int32
+        key :required, false
+      end
+
+      parameter do
+        key :name, :offset
+        key :in, :query
+        key :description, 'The number of session events the query will begin reading from the start of the set. (Default: 0)'
+        key :example, 0
+        key :type, :integer
+        key :format, :int32
+        key :required, false
+      end
+
+      parameter do
+        key :name, :order
+        key :in, :query
+        key :description, 'The session event created_at sort order. (Default: desc)'
```

Comment:
Going off the above statement of making `ORDER_ENUM` a re-usable variable, we could probably make this description more generic and also put it in `RootApiDoc`. Something like `The order in which to display the results, based on the created_at time. (Default: desc)`.